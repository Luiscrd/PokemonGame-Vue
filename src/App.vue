<template>
  <Header />
  <PokemonPage />
  <Footer />
</template>

<script>
import Header from './components/shared/Header.vue'
import Footer from './components/shared/Footer.vue'
import PokemonPage from './pages/PokemonPage.vue'

export default {
  name: 'App',
  components: {
    Header,
    PokemonPage,
    Footer
  }
}
</script>

<style>
html, body{
  margin: 0;
  padding: 0;
}

body{
  font-family: Arial, Helvetica, sans-serif;
  position: relative;
  display: flex;
  flex-direction: column;
  min-height: 100vh;
}

#app{
  position: relative;
  display: flex;
  flex-direction: column;
  min-height: 100vh;
}
</style>
