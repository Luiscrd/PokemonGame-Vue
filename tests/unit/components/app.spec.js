import { shallowMount } from '@vue/test-utils'
import App from '@/App.vue'

describe('App Component', () => {

    let wrapper;

    beforeEach(() => {

        wrapper = shallowMount(App)

    });

    test('Debe de hacer mach con el snapshot', () => {

        expect(wrapper.html()).toMatchSnapshot()

    })

})