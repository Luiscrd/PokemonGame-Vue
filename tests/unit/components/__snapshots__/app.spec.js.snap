// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Debe de hacer mach con el snapshot 1`] = `
<header-stub></header-stub>
<pokemon-page-stub></pokemon-page-stub>
<footer-stub></footer-stub>
`;
